invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/side_bar/external_resources.html

```diff
@@ -0,0 +1,59 @@
+{#
+Copyright (C) 2020 CERN.
+
+Invenio RDM Records is free software; you can redistribute it and/or modify
+it under the terms of the MIT License; see LICENSE file for more details.
+#}
+{% set external_resources = record["ui"].get("related_identifiers", None) %}
+
+{%- if external_resources -%}
+  <section id="external-resource" aria-label="{{ _('External resources') }}" class="ui segment rdm-sidebar external resource">
+  <h2 class="ui small header">{{ _('External resources')}}</h2>
+  {# dynamically create the list of external resources #}
+  <ul class="details-list m-0 p-0" style="list-style-type: none;">
+    {%- for itr in config.get("APP_RDM_RECORD_LANDING_PAGE_EXTERNAL_LINKS") -%}
+      {%- for resource_itr in external_resources if resource_itr['scheme'] == 'url' and itr['id'] in resource_itr.get('identifier', []) -%}
+
+        {% set resource = itr.render(resource_itr) %}
+
+        {%- if resource -%}
+          {%- if resource.template -%}
+            {# It's up to the template to render the `resource` as it wishes #}
+            {%- include resource.template %}
+          {%- else -%}
+            {# Default rendering #}
+            <li class="has-popup">
+            <div class="resource clickable" tabindex="0" aria-haspopup="dialog"
+              aria-expanded="false" role="button" aria-label="{{ resource.content.title }}"
+              style="display:flex; flex-direction: row;">
+
+              {% if resource.content.icon %}
+                {% set iconFile = 'images/{}'.format(resource.content.icon) %}
```

Comment:
I would check if the path is accessible by some variables in jinja (there was a way to access statics if I remember correctly), instead of hardcoding it

Reply:
Do you mean the 
`url_for`
method?
Like this:

` <img class="icon" src="{{ url_for('static', filename=iconFile) }}" alt="{{ license.id }} icon"/>`

Reply:
please check: https://github.com/inveniosoftware/invenio-theme/blob/master/invenio_theme/templates/semantic-ui/invenio_theme/header.html#L25